src/helpers/Constant.ts

```diff
@@ -0,0 +1,6 @@
+export const enum roleType {
```

Comment:
Please use PascalCase for enum names, refer to [styleguide](https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#enum)

Rename `roleType` to `RoleType`.